openedx/core/djangoapps/oauth_dispatch/tests/test_jwt.py

```diff
@@ -163,3 +165,33 @@ def test_scopes(self):
         assert jwt_payload['scopes'] == self.default_scopes
         assert jwt_scopes_payload['scopes'] == scopes
         assert jwt_scopes_payload['user_id'] == self.user.id
+
+    def test_password_grant_type(self):
+        oauth_adapter = DOTAdapter()
+        token_dict = self._get_token_dict(client_restricted=False, oauth_adapter=oauth_adapter)
+        jwt_token_dict = jwt_api.create_jwt_token_dict(token_dict, oauth_adapter, use_asymmetric_key=False)
+
+        self.assert_valid_jwt_access_token(
+            jwt_token_dict["access_token"], self.user, self.default_scopes,
+            grant_type=dot_models.Application.GRANT_PASSWORD,
+        )
+
+    def test_empty_str_grant_type(self):
```

Comment:
@jawad-khan: Can you let me know if you added tests for #2, or tested separately, or what happened. Thanks. I need to run. Maybe this will be obvious when I complete the review anyway.

Reply:
Maybe I am not getting your point clear but you can correct me. I did some testing and used other grant types and got expected grant types in return.